x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/inference/ltr/LearningToRankRescorerBuilderRewriteTests.java

```diff
@@ -218,6 +218,24 @@ public void testBuildContext() throws Exception {
         );
     }
 
+    public void testBuildContextWindowSize() throws IOException {
```

Comment:
Should we have additional tests with invalid window size data?

Reply:
These tests are already present in the code for rescorer as it is shared concerns.